packages/gatsby/src/redux/persist.ts

```diff
@@ -43,7 +43,8 @@ function reduxWorkerSlicesPrefix(dir: string): string {
 }
 
 export function readFromCache(
-  slices?: Array<GatsbyStateKeys>
+  slices?: Array<GatsbyStateKeys>,
+  optionalPrefix?: string
```

Comment:
Shouldn't this have a default value? Otherwise, it will be converted to `undefined` in the filename when omitted?

```suggestion
  optionalPrefix?: string = ``
```